### PR TITLE
Implement community board (organization tests) [4/7]

### PR DESCRIPTION
**Pull request by isaachan100**
## Overview

Implemented jest testing for community board organizations

## Changes Made

- Implemented unit jest test cases for organizations
- onboarded 3 new organizations with organizations.json
- added flyer queries/mutations and organization queries to app.ts

## Test Coverage

- verified organization queries locally
- organizations were successfully added to local database
- jest testing runs to completion

## Next Steps (delete if not applicable)

- will implement user routes, categories for organizations, and onboard remaining organizations

### CODE REVIEW

File: src/tests/organization.test.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    await FlyerModel.insertMany(flyers);
+
+    const getOrganizationsResponse = await OrganizationRepo.getMostRecentFlyer(pub);
+    const respDate = new Date(getOrganizationsResponse.date);
+
+    const flyerDates = FactoryUtils.mapToValue(flyers, 'date');
+
+    const isMin = flyerDates.every((d) => {
+      return respDate.getTime() >= new Date(d).getTime();
+    });
```
Comment (snajima) [inline src/tests/organization.test.ts:57]: whoa this is pretty cool

**Review by snajima — APPROVED** [review]
lgtm! v prod

File: organizations.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+	"organizations": [{
+			"bio": "",
+			"name": "Women in Stem at Cornell",
```
Comment (katesliang) [inline organizations.json:4]: WOMEN IN STEM

**Review by katesliang — APPROVED** [review]
prodge

File: src/tests/organization.test.ts (hunk 1 above)
Comment (isaachan100) [reply src/tests/organization.test.ts:57]: XD it was kidus' idea

File: organizations.json (hunk 2 above)
Comment (isaachan100) [reply organizations.json:4]: THATS YOU SO PROD

**Pull request merged**